satellite/audit/reporter.go

```diff
@@ -148,6 +149,7 @@ func (reporter *Reporter) recordOfflineStatus(ctx context.Context, offlineNodeID
 		}
 	}
 	if len(failed) > 0 {
+		reporter.log.Debug("Error recording Audit Offline Nodes ", zap.Strings("Node IDs", failed.Strings()))
```

Comment:
This logger is named `audit:reporter`, so having the word "Audit" in the message might be unnecessary

Reply:
updated